server/__tests__/integration/v1.campaigns.test.js

```diff
@@ -0,0 +1,36 @@
+// below are where the tests for the campaign endpoints will go, I kept getting errors back and couldnt set this up correctly 😞, i got thrown off by the section about inserting test data into the database
```

Comment:
Don't worry! We can look at your testing issues during Pair Hours.

Reply:
Sounds good. I missed yesterdays standup so hopefully we can take a look at it at the next Pair Hours.